### DISCUSSION TITLE
Can you embed the _internal directory inside the executable?

### DISCUSSION THREAD

**Discussion by yesseruser** [pyinstaller/pyinstaller#8921] [Help]
Hi, I'm wondering if pyinstaller has some sort of single-file mode where all required files are within the executable itself. I know .NET has this and I'd like to know if pyinstaller has this feature too.

**Comment by bwoodsend** [comment]
Ermm, I'm pretty certain it would have taken you less time to search the documentation for the words `one file` than it would to have asked this question.

**Reply by yesseruser** [reply]
Yeah, I found it afterwards but forgot to write here. Thank you.
